theme/base-2016/partials/_footer.twig

```diff
@@ -17,6 +17,8 @@
 
 </footer>
 
-<script src="{{ asset('js/jquery.min.js', 'theme') }}"></script>
+{# Extensions and copypasta gleaned from other sources tend to cause multiple jQuery
+   includes. We only include it here conditionally, if not done so already. #}
+<script>window.jQuery || document.write("<script src='{{ asset('js/jquery.min.js', 'theme') }}'><\/script>");</script>
```

Comment:
@cdowdy I am happy you're letting us know, and i'll happily consider alternatives.

I'm just not convinced that this is a problem yet. It says _all_ criteria need to be met:

> Specifically Chrome will not execute the <script> elements injected via document.write() when **all** of the following conditions are met:

If you have an alternative solution to this problem, i'm all ears.